tests/TestCase/Controller/ModulesControllerTest.php

```diff
@@ -607,6 +609,104 @@ public function testRelationshipsJson(string $relation, string $objectType) : vo
         $this->assertExpectedViewVars(['_serialize']);
     }
 
+    /**
+     * Data provider for `testGetThumbsUrls` test case.
+     *
+     * @return array
+     */
+    public function getThumbsUrlsProvider() : Array
+    {
+        return [
+            // test with empty object
+            'emptyResponse' => [
+                [],
+                [],
+            ],
+            // test with objct without ids
+            'responseWithoutIds' => [
+                ['data' => []],
+                ['data' => []],
+            ],
+            // correct result
+            'correctResponseMock' => [
+                [ // expected
+                    'data' => [
+                        [
+                            'id' => '43',
+                            'meta' =>
+                                [
+                                    'url' => 'https://s3-eu-west-1.amazonaws.com/be4-media-test/test-thumbs/472f9968-b02e-4253-b65f-0bb95f02b89c-test.png/728b91672d309feb717f12aede127da99b1e47b9.png',
```

Comment:
[minor] since it's a mock I'd use `example.com` domain, something like `https://media.example.com/thumbs/.....`
to avoid domains actually in use..

Reply:
ok